roles/kubernetes/preinstall/templates/openstack-cloud-config.j2

```diff
@@ -15,12 +15,22 @@ domain-id ="{{ openstack_domain_id }}"
 bs-version={{ openstack_blockstorage_version }}
 {% endif %}
 
-{% if openstack_lbaas_enabled and openstack_lbaas_subnet_id %}
+{% if openstack_lbaas_enabled and openstack_lbaas_subnet_id is defined %}
 [LoadBalancer]
 subnet-id={{ openstack_lbaas_subnet_id }}
 {% if openstack_lbaas_floating_network_id is defined %}
 floating-network-id={{ openstack_lbaas_floating_network_id }}
 {% endif %}
+{% if openstack_lbaas_use_octavia is defined %}
```

Comment:
I think we need to move this above line 23. I don't think you can use-octavia without [LoadBalancer] section being created.

Reply:
Yes but it is already inside the first if so it will be created unless `openstack_lbaas_enabled and openstack_lbaas_subnet_id is defined` condition is met

Reply:
oh nevermind it's ok